src/main/resources/META-INF/plugin.xml

```diff
@@ -141,6 +141,7 @@
     <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html
          on how to target different products -->
     <depends>com.intellij.modules.lang</depends>
+    <depends>Git4Idea</depends>
```

Comment:
Shall we make it optional already in this PR, or later when working on the events?